runit/web-sharelatex/run

```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
 export SHARELATEX_CONFIG=/etc/sharelatex/settings.coffee
-exec /sbin/setuser www-data /usr/bin/node /var/www/sharelatex/web/app.js >> /var/log/sharelatex/web.log 2>&1
\ No newline at end of file
+
+NODE_PARAMS=""
+if [ "$DEBUG_NODE" == "true" ]; then
+    echo "running debug - web"
+    NODE_PARAMS="--inspect=0.0.0.0:40000"
```

Comment:
```suggestion
    NODE_PARAMS="--inspect=0.0.0.0:30000"
```

30000 would be consistent with the other ports

Reply:
40000 for `web` is consistent with existing tooling already set in place.

Reply:
That's internal I guess based on [1]?

---
[1] https://github.com/search?l=&q=4000+user%3Aoverleaf